crates/agglayer-node/src/node.rs

```diff
@@ -326,6 +353,49 @@ impl Node {
         Ok(node)
     }
 
+    fn serve_rpc(
+        name: &str,
+        handles: &mut Vec<axum_server::Handle>,
+        addr_plain: SocketAddr,
+        addr_tls: SocketAddr,
```

Comment:
Should both of this endpoints be optional, and handle plain, tls or both endpoints depending of the passed arguments.